### ISSUE TITLE
the output is strange after training on my own dataset

### ISSUE THREAD

**Issue by milk-bottle-liyu** [driving-behavior/DBNet#7]
Hi,friend, it's me again. <*_*>
I use my own data to train my network. Because I only recorded the steer angle, so I modified it to output only one float, the steer angle. However something strange happened. The mean loss in training stage stays high. Then I check the output on the val dataset and find the values are all positive. Though my dataset is not as big as yours, it still contains thousands samples from 4 scenes.  Some of my outputs are in the picture. I tried to normalized the data, but it did not work. What should I do to solve the problem?
![image](https://user-images.githubusercontent.com/17823646/48963573-0b806500-efd0-11e8-9444-2152a84c29bc.png)

**Comment by wangjksjtu** [comment]
Hi, it is pretty strange. There must be some bugs while transplanting to your dataset. I could not ensure the problems with the information provided. Have you modified the loss by the way or adjust the hyper-parameters? Will the training loss decrease in the training?

**Comment by milk-bottle-liyu** [comment]
Sorry for not responding in time. I don`t understand why should I change the loss? For now I am working on the image only model, and besides IO I did not change any other things. The loss only drops in the first epoch and did not decrease over dozens epochs. The loss is like:
![image](https://user-images.githubusercontent.com/17823646/49205094-e719f380-f3e8-11e8-9847-d3c454f8bea3.png)

What can I do to make the loss decrease? I had doubled the train set already.
